### PR TITLE
add justfile and improve readme

### PR DESCRIPTION
**Pull request by Catrya**
## Release Notes

- **New Features**
	- Enhanced documentation in the README files, including sections for "Protocol" and "Users-guide" to assist contributors and general users.
	- Added commands for project setup and management using the Just tool, including `just init`, `just serve`, and `just build`.
	- Introduced Spanish-language README and commands for the users' guide.

- **Documentation**
	- Improved clarity and accessibility of project documentation for both English and Spanish-speaking users.

### CODE REVIEW

File: protocol/justfile [hunk 1]
```diff
@@ -0,0 +1,9 @@
+init:
+  cargo install mdbook --version 0.4.37
+  cargo install mdbook-admonish --version 1.15.0
+
+build:
+  mdbook build
```
[the remark this answers is not in the document]
Comment (grunch) [reply protocol/justfile]: looks fair, this rabbit knows his thing...

File: protocol/justfile [hunk 2]
```diff
@@ -0,0 +1,9 @@
+init:
+  cargo install mdbook --version 0.4.37
+  cargo install mdbook-admonish --version 1.15.0
```
[the remark this answers is not in the document]
Comment (grunch) [reply protocol/justfile]: I agree with the 🐰

File: protocol/justfile [hunk 3]
```diff
@@ -0,0 +1,9 @@
...
+
+build:
+  mdbook build
+
+serve:
+  mdbook serve
```
[the remark this answers is not in the document]
Comment (grunch) [reply protocol/justfile]: rabbit can I be your friend?

**Comment by Catrya** [comment]
thanks to the 🐰 :)

**Comment by Catrya** [comment]
@grunch en el commit 5f7d3b2...  ya modifiqué el justfile según las recomendaciones del CodeRabbit, pero va a seguir proponiendo mejoras, creo que ya está bien así. Los readme sí los volví a modificar con las ultimas recomendaciones

**Pull request merged**